components/text/sed/test/results-64.master

```diff
@@ -0,0 +1,252 @@
+PASS: test-set-mode-acl-2.sh
+PASS: test-copy-acl.sh
+PASS: test-copy-acl-1.sh
+PASS: test-copy-acl-2.sh
+PASS: test-alignof
+PASS: test-alloca-opt
+PASS: test-binary-io.sh
+PASS: test-btowc1.sh
+PASS: test-btowc2.sh
+PASS: test-c-ctype
+PASS: test-c-strcase.sh
+PASS: test-canonicalize-lgpl
+PASS: test-chdir
+PASS: test-close
+PASS: test-ctype
+PASS: dfa-invalid-char-class.sh
+PASS: dfa-match.sh
+PASS: test-dirent
+PASS: test-dup2
+PASS: test-environ
+PASS: test-errno
+PASS: test-fcntl-h
+PASS: test-fdopen
+PASS: test-fflush
+PASS: test-fflush2.sh
+PASS: test-fgetc
+PASS: test-file-has-acl.sh
+PASS: test-file-has-acl-1.sh
+PASS: test-file-has-acl-2.sh
+PASS: test-fpending.sh
+PASS: test-fpurge
+PASS: test-fputc
+PASS: test-fread
+PASS: test-freading
+PASS: test-fseek.sh
+PASS: test-fseek2.sh
+PASS: test-fseeko.sh
+PASS: test-fseeko2.sh
+PASS: test-fseeko3.sh
+PASS: test-fseeko4.sh
+PASS: test-fstat
+PASS: test-ftell.sh
+PASS: test-ftell2.sh
+PASS: test-ftell3
+PASS: test-ftello.sh
+PASS: test-ftello2.sh
+PASS: test-ftello3
+PASS: test-ftello4.sh
+PASS: test-fwrite
+PASS: test-fwriting
+PASS: test-getcwd-lgpl
+PASS: test-getdelim
+PASS: test-getdtablesize
+PASS: test-getopt-gnu
+PASS: test-getopt-posix
+PASS: test-getprogname
+PASS: test-gettimeofday
+PASS: test-ignore-value
+PASS: test-intprops
+PASS: test-inttypes
+PASS: test-isblank
+PASS: test-langinfo
+PASS: test-limits-h
+PASS: test-link
+PASS: test-locale
+PASS: test-localeconv
+FAIL: test-localename
+PASS: test-lseek.sh
+PASS: test-lstat
+PASS: test-malloca
+PASS: test-mbrtowc1.sh
+PASS: test-mbrtowc2.sh
+SKIP: test-mbrtowc3.sh
+SKIP: test-mbrtowc4.sh
+PASS: test-mbrtowc5.sh
+SKIP: test-mbrtowc-w32-1.sh
+SKIP: test-mbrtowc-w32-2.sh
+SKIP: test-mbrtowc-w32-3.sh
+SKIP: test-mbrtowc-w32-4.sh
+SKIP: test-mbrtowc-w32-5.sh
+PASS: test-mbsinit.sh
+PASS: test-memchr
+PASS: test-memrchr
+PASS: test-nl_langinfo.sh
+PASS: test-open
+PASS: test-pathmax
+PASS: test-quotearg-simple
+PASS: test-read-file
+PASS: test-readlink
+PASS: test-regex
+PASS: test-rename
+PASS: test-rmdir
+PASS: test-setenv
+PASS: test-setlocale1.sh
+PASS: test-setlocale2.sh
+PASS: test-stat
+PASS: test-stdalign
+PASS: test-stdbool
+PASS: test-stddef
+PASS: test-stdint
+PASS: test-stdio
+PASS: test-stdlib
+PASS: test-strerror
+PASS: test-string
+PASS: test-strverscmp
+PASS: test-symlink
+PASS: test-sys_stat
+PASS: test-sys_time
+PASS: test-sys_types
+PASS: test-init.sh
+PASS: test-time
+PASS: test-unistd
+PASS: test-unsetenv
+PASS: test-update-copyright.sh
+PASS: test-vc-list-files-git.sh
+SKIP: test-vc-list-files-cvs.sh
+PASS: test-verify
+PASS: test-verify.sh
+PASS: test-version-etc.sh
+PASS: test-wchar
+PASS: test-wcrtomb.sh
+SKIP: test-wcrtomb-w32-1.sh
+SKIP: test-wcrtomb-w32-2.sh
+SKIP: test-wcrtomb-w32-3.sh
+SKIP: test-wcrtomb-w32-4.sh
+SKIP: test-wcrtomb-w32-5.sh
+PASS: test-wctype-h
+PASS: test-xalloc-die.sh
+# TOTAL: 130
+# PASS:  116
+# SKIP:  13
+# XFAIL: 0
+# FAIL:  1
+# XPASS: 0
+# ERROR: 0
```

Comment:
Tests depend on locale/fr-extra , developer/debug/valgrind (and perhaps locale/fr). Better add them to REQUIRED_PACKAGES with corresponding comments.